README.md

```diff
@@ -185,6 +185,14 @@ Note that although the annotation containing router configuration for each of th
 | deis-router          | workerProcesses  | `string`   | `auto` (number of CPU cores) | Number of worker processes to start. |
 | deis-router          | workerConnections | `integer` | `768`         | Maximum number of simultaneous connections that can be opened by a worker process. |
 | deis-router          | defaultTimeout   | `integer`  | 1300          | Default timeout value in seconds.  Should be greater than the front-facing load balancer's timeout value. |
+| deis-router          | useGzip          | `boolean`  | `true`        | Whether to use gzip compression. |
```

Comment:
is it a limitation in `confd` that wouldn't allow you to test if `gzipConfig` is empty or not and thus apply whichever of the config elements?